torch/onnx/diagnostic/__init__.py

```diff
@@ -0,0 +1,14 @@
+from ._diagnostic import Diagnostic, Level, Rule
```

Comment:
Should this be in `_internal`?

Reply:
Good idea, there has been some big changes in structure which will move a large portion of stuff to there. A quick question while we are on this. Is there a preference between `_internal._diagnostic/**.py` vs `_internal.diagnostic/**.py`? I noticed the `_beartype.py`  still has a underscore despite it being in internal folder already.

Reply:
_internal.diagnostic to me is preferable for its simplicity. _beartype was there to avoid redefining the beartype package's namespace